src/utils/rem.ts

```diff
@@ -0,0 +1,35 @@
+export enum DeviceSize {
+    PC = 1440,
+    WAP = 393,
+}
+
+/**
+ * @description Get the current screen-to-design ratio.
+ */
+export const getClientRatio = () => {
+    const clientWidth = window.innerWidth;
+    let base = DeviceSize.WAP;
+    if (clientWidth >= 768) {
+        if (clientWidth > DeviceSize.PC && clientWidth <= 2560) {
+            return 1;
+        }
+        base = DeviceSize.PC;
+    }
+    return Math.min(clientWidth / base, 2);
+};
+
+
+/**
+ * @description Get the rem ratio.
+ */
+export const getRemRatio = () => {
+    return 1 / 16;
+};
+
+/**
+ * @description Convert px to rem.
+ */
+export const px2rem = (value: number, hasUnit = true) => {
```

Comment:
Where is this function applied?

Reply:
The `postcss-pxtorem` plugin operates at the compilation stage and can only compile static classes or CSS. Sometimes, if you need to calculate or dynamically set a px value in JavaScript, you'll need to convert it to rem manually.